### PR TITLE
Update support thai languages.

### PR DESCRIPTION
**Pull request by DekThaiInChina**
Add thai language

### CODE REVIEW

**Comment by konstrybakov** [comment]
Hey @DekThaiInChina! Thank you for your contribution! Is this PR relevant knowing that there is this PR from you also: https://github.com/toptal/gitignore.io/pull/601/files?

**Comment by DekThaiInChina** [comment]
@konstrybakov Such PR is related. Just that I have separated for ease of my work.

File: Sources/App/Extensions/Router+Extensions.swift [hunk 1]
```diff
@@ -17,7 +17,7 @@ extension Request {
                 .first?
                 .split(separator: "-")
                 .first ?? "en")
-            let supportedLanguages = ["ar", "de", "en", "fa", "fr", "id", "ja", "ko", "pt", "ro", "ru", "tr", "zh", "nl", "el"]
+            let supportedLanguages = ["ar", "de", "en", "fa", "fr", "id", "ja", "ko", "pt", "ro", "ru", "tr", "zh", "nl", "el", "th"]
```
Comment (filipechagas) [inline Sources/App/Extensions/Router+Extensions.swift:20]: Isn't this the exact same change as https://github.com/toptal/gitignore.io/pull/601/files#diff-f088ce64e1518b25971bf76a0a936cbd287599e47030a26dc5d77ba51f7bda43R20 ?
Why do we need both PRs?

**Review by filipechagas — CHANGES_REQUESTED** [review]
Hey @DekThaiInChina 
Thank you for contributing

I'd like to reinforce what @konstrybakov mentioned. It seems like you've repeated the changes from https://github.com/toptal/gitignore.io/pull/601

File: Sources/App/Extensions/Router+Extensions.swift (hunk 1 above)
Comment (DekThaiInChina) [reply Sources/App/Extensions/Router+Extensions.swift:20]: Thanks for suggestion #602, I should have done it twice due to a problem with my internet.

**Comment by filipechagas** [comment]
Closing it as it duplicates https://github.com/toptal/gitignore.io/pull/601

**Pull request closed without merging**